src/cbor.c

```diff
@@ -1262,12 +1302,16 @@ cbor_decode_cred_authdata(const cbor_item_t *item, int cose_alg,
 }
 
 int
-cbor_decode_assert_authdata(const cbor_item_t *item, fido_blob_t *authdata_cbor,
-    fido_authdata_t *authdata, int *authdata_ext, fido_blob_t *hmac_secret_enc)
+cbor_decode_assert_authdata(const cbor_item_t *item, fido_assert_stmt *stmt)
```

Comment:
should we explore grouping `hmac_secret_enc` and `blob` in a new `struct fido_assert_extattr` and pass the latter here instead of giving this function access to the whole assertion statement?

Reply:
fixed in 7cc5c588237a9e256eb8f06b6a6c257d93e5e71c, may need some additional touchups